config/default.json

```diff
@@ -89,6 +89,7 @@
     "MAX_BYTE_SIZE": 10000000,
     "EXPIRES_IN_MS": 1000
   },
+  "API_DOCUMENTATION_ENDPOINTS": true,
```

Comment:
Please add a `_ENABLED` prefix, perhaps something like this:
`API_DOCUMENTATION_ENDPOINTS` --> `API_DOC_ENDPOINTS_ENABLED`